### ISSUE TITLE
Cope with Slack timing out

### ISSUE THREAD

**Issue by palfrey** [palfrey/mopidy-tachikoma#5]
Exceptions like the following

```
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: Unhandled exception in thread started by <bound method TachikomaFrontend.doSlack of <mopidy_tachikoma.bot.TachikomaFrontend object at 0x7456aff0>>
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: Traceback (most recent call last):
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/mopidy_tachikoma/bot.py", line 34, in doSlack
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: items = self.sc.rtm_read()
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/slackclient/_client.py", line 120, in rtm_read
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: json_data = self.server.websocket_safe_read()
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/slackclient/_server.py", line 140, in websocket_safe_read
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: data += "{0}\n".format(self.websocket.recv())
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 298, in recv
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: opcode, data = self.recv_data()
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 315, in recv_data
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: opcode, frame = self.recv_data_frame(control_frame)
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 328, in recv_data_frame
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: frame = self.recv_frame()
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 360, in recv_frame
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: return self.frame_buffer.recv_frame()
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/websocket/_abnf.py", line 312, in recv_frame
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: self.recv_header()
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/websocket/_abnf.py", line 261, in recv_header
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: header = self.recv_strict(2)
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/websocket/_abnf.py", line 346, in recv_strict
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: bytes = self.recv(min(16384, shortage))
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 429, in _recv
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: return recv(self.sock, bufsize)
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/local/lib/python2.7/dist-packages/websocket/_socket.py", line 77, in recv
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: bytes = sock.recv(bufsize)
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/lib/python2.7/ssl.py", line 714, in recv
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: return self.read(buflen)
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: File "/usr/lib/python2.7/ssl.py", line 608, in read
Aug 03 22:06:02 mopidy-jukebox mopidy[26890]: v = self._sslobj.read(len or 1024)
```

**Comment by palfrey** [comment]
Very similar
```
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: Unhandled exception in thread started by <bound method TachikomaFrontend.doSlack of <mopidy_tachikoma.bot.TachikomaFrontend object at 0x74497fd0>>
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: Traceback (most recent call last):
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/mopidy_tachikoma/bot.py", line 34, in doSlack
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: items = self.sc.rtm_read()
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/slackclient/_client.py", line 120, in rtm_read
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: json_data = self.server.websocket_safe_read()
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/slackclient/_server.py", line 140, in websocket_safe_read
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: data += "{0}\n".format(self.websocket.recv())
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 298, in recv
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: opcode, data = self.recv_data()
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 315, in recv_data
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: opcode, frame = self.recv_data_frame(control_frame)
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 328, in recv_data_frame
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: frame = self.recv_frame()
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 360, in recv_frame
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: return self.frame_buffer.recv_frame()
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/websocket/_abnf.py", line 312, in recv_frame
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: self.recv_header()
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/websocket/_abnf.py", line 261, in recv_header
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: header = self.recv_strict(2)
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/websocket/_abnf.py", line 346, in recv_strict
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: bytes = self.recv(min(16384, shortage))
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/websocket/_core.py", line 429, in _recv
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: return recv(self.sock, bufsize)
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: File "/usr/local/lib/python2.7/dist-packages/websocket/_socket.py", line 89, in recv
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: raise WebSocketConnectionClosedException("Connection is already closed.")
Aug 12 12:07:06 mopidy-jukebox mopidy[677]: websocket._exceptions.WebSocketConnectionClosedException: Connection is already closed.
```

**Comment by palfrey** [comment]
Fixed by 7483954624365a39c8ada7c2939e8901eef63cdd
